contracts/stellar-interchain-token-service/src/tests/utils/mod.rs

```diff
@@ -70,97 +49,32 @@ pub fn setup_env<'a>() -> (
     (env, client, gateway_client, gas_service_client, signers)
 }
 
-pub fn setup_migrate_env<'a>() -> MigrateTestConfig<'a> {
-    let (env, its_client, ..) = setup_env();
-    let upgrader_client = setup_upgrader(&env);
-    let owner: Address = its_client.owner();
-    let (token_id, _) = setup_its_token(&env, &its_client, &owner, 100);
-
-    let its_wasm_hash = env
-        .deployer()
-        .upload_contract_wasm(NEW_INTERCHAIN_TOKEN_SERVICE_WASM);
-    let new_token_manager_wasm_hash = env.deployer().upload_contract_wasm(TOKEN_MANAGER_WASM_V110);
-    let new_interchain_token_wasm_hash = env
-        .deployer()
-        .upload_contract_wasm(INTERCHAIN_TOKEN_WASM_V110);
+pub fn format_auths(
+    auths: std::vec::Vec<(Address, AuthorizedInvocation)>,
```

Comment:
```suggestion
    env: &Env,
```
call `env.auths()` internally perhaps